.yamato/config/abv.metafile

```diff
@@ -0,0 +1,95 @@
+project:
+  name: ABV
+
+jobs:
+  - name: Nightly
+    dependencies:
+      - .yamato/all-hdrp.yml#Nightly_HDRP_<TRACK>
+      - .yamato/all-vfx.yml#Nightly_VFX_<TRACK>
+      - .yamato/all-urp.yml#Nightly_URP_<TRACK>
+      - .yamato/all-shadergraph.yml#Nightly_ShaderGraph_<TRACK>
+      - <PROJECT_ALL_YML_FILE>#General_Checks_<TRACK>
```

Comment:
What does PROJECT_ALL_YML_FILE refer to? (where is it set?)

Reply:
It refers to `.yamato/all-{project}.yml` for the corresponding metafile (the abv metafile works similar to projects now, but is a specific case and outputs to `.yamato/_abv.yml` still), that comes with https://github.cds.internal.unity3d.com/unity/gfx-sdet-tools/pull/43 . I can see the confusion though, I'm fine with changing this to the concrete path directly

Reply:
I see, that's fine don't need to worry about updating both PRs